data/sql_updates/totals_candidate_committee.sql

```diff
@@ -118,36 +93,42 @@ with last_cycle as (
         sum(p.tranf_to_other_auth_cmte) as transfers_to_other_authorized_committee,
         sum(p.net_op_exp) as net_operating_expenditures,
         sum(p.net_contb) as net_contributions,
-        -- these are added in the event that a candidate has multiple committees
-        false as full_election,
-        coalesce(min(first.cvg_start_dt), min(last.coverage_start_date)) as coverage_start_date,
-        sum(last.cash_on_hand_end_period) as last_cash_on_hand_end_period,
-        max(last.report_type_full) as last_report_type_full,
-        sum(last.debts_owed_to_committee) as last_debts_owed_to_committee,
-        sum(last.debts_owed_by_committee) as last_debts_owed_by_committee,
-        max(last.beginning_image_number) as last_beginning_image_number,
-        max(last.report_year) as last_report_year,
-        min(first.cash_on_hand_beginning_of_period) as cash_on_hand_beginning_of_period
+        false as full_election
+
     from
-        -- starting with candidate will consolidate record in the event that a candidate has multiple committees
         ofec_cand_cmte_linkage_mv_tmp link
         left join disclosure.v_sum_and_det_sum_report p on link.cmte_id = p.cmte_id and link.fec_election_yr = get_cycle(p.rpt_yr)
-        left join last_cycle last on link.cmte_id = last.cmte_id and link.fec_election_yr = last.cycle
-        left join first_cycle first on link.cmte_id = first.committee_id and link.fec_election_yr = first.cycle
     where
         link.fec_election_yr >= :START_YEAR
-        and p.form_tp_cd = 'F3P'
+        -- this issue with the data is really driving me nuts.  Jeff said he's looking into it,
+        -- leaving a reminder here, but this check filters out these 9999... records.
+        and p.cvg_start_dt != 99999999
```

Comment:
I am concerned that this might omit records and cause the totals to be wrong. Though getting rid of them may be the best solution for the data we have. I am wondering if we can we filter them out of the first and last sections and keep them in the cycle totals?